init/common.js

```diff
@@ -162,28 +167,31 @@ const createInterests = async (userId) => {
   return interestId;
 };
 
-const createProject = async (interestId, userId) => {
-  const service = strapi.service("api::project.project");
-  let existing = await service.findOne({});
+const createProject = async (interestId, userId, project, addTeam=false) => {
+  let existing = await strapi.entityService.findMany("api::project.project", {filters: {slug: project.slug}});
+  existing = existing[0];
   if (existing) {
     return;
   }
-  config.project.interests = [interestId];
-  config.project.team = {
-    members: [
-      {
-        member: userId,
-        role: "Developer",
-      },
-    ],
-    leaders: [
-      {
-        leader: userId,
-        role: "Project Manager",
-      },
-    ],
-  };
-  existing = await service.create({ data: config.project });
+  project.interests = [interestId];
+  if(!addTeam){
```

Comment:
Probably need to reverse logic here

Reply:
@nikita-pardeshi-github I think Wolfgang means this should be `if(addTeam)`.